### DISCUSSION TITLE
Excel import - bigger size [Sep 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6608] [Discord Threads]
> 🙋‍♂️ **greg.87**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### One option that i could quickly think of, is to break down file into two. Import one & use upload as CSV for the next  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1146547607041224774)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
